ecosystem/python/sdk/Makefile

```diff
@@ -5,9 +5,9 @@ test:
 	- poetry run python -m unittest discover -s aptos_sdk/ -p '*.py' -t ..
 
 fmt:
-	- find . -type f -name "*.py" | xargs poetry run autoflake -i -r --remove-all-unused-imports --remove-unused-variables --ignore-init-module-imports
-	- find . -type f -name "*.py" | xargs poetry run isort
-	- find . -type f -name "*.py" | xargs poetry run black
+	- find ./examples ./aptos_sdk -type f -name "*.py" | xargs poetry run autoflake -i -r --remove-all-unused-imports --remove-unused-variables --ignore-init-module-imports
+	- find ./examples ./aptos_sdk -type f -name "*.py" | xargs poetry run isort
+	- find ./examples ./aptos_sdk -type f -name "*.py" | xargs poetry run black
```

Comment:
we should have probably added setup.py to this

Reply:
yeah. I will open another pull request to do this